templates/pricing/desktop.html

```diff
@@ -74,176 +71,326 @@ <h2 class="p-heading--4">Additional benefits for Enterprise Services Plus</h2>
+              </td>
+            </tr>
+            <tr>
+              <th scope="row">Snap Store configuration</th>
+              <td aria-label="Not available">
+                <span class="u-text--muted">&ndash;</span>
+              </td>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">
+                  Snap Store proxy deployment and local snap package repository.
+                </p>
+              </td>
+            </tr>
+            <tr>
+              <th scope="row">CIFS/SMB configuration</th>
+              <td aria-label="Not available">
+                <span class="u-text--muted">&ndash;</span>
+              </td>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">
+                  Workstations configured to mount CIFS/SMB file systems from a third-party source
+                </p>
+              </td>
+            </tr>
+            <tr>
+              <th scope="row" rowspan="4">Security Configuration</th>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">Installed software audit</p>
+              </td>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">Installed software audit</p>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">Security policy enforcement</p>
+              </td>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">Custom firewall enablement</p>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">Anti-virus/anti-malware solution support</p>
+              </td>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">Security policy enforcement</p>
+              </td>
+            </tr>
+            <tr>
+              <td role="presentation"></td>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">Security policy enforcement</p>
+              </td>
+            </tr>
+            <tr>
+              <th scope="row">Encrypted Volume Support</th>
+              <td aria-label="Not available">
+                <span class="u-text--muted">&ndash;</span>
+              </td>
+              <td>
+                <p class="u-text--muted u-no-margin--bottom u-no-padding--top">Third party encryption support</p>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+        <div class="row">
+          <div class="col-start-large-4 col-6">
+            <div class="p-cta-block">
+              <a class="p-button"
+                 href="https://assets.ubuntu.com/v1/a48fc9cd-Ubuntu%20Desktop%20DS%2021.06.23.pdf">Download the datasheet</a>
```

Comment:
The datasheet link does not match the copy docs, I think the copy doc link is outdated. If that's the case can you suggest to update the copy doc link please?